engine/security/trust/trust_automation.md

```diff
@@ -75,6 +75,7 @@ you should set the environment variable `DOCKER_CONTENT_TRUST` either manually o
 in a scripted fashion. Consider the simple Dockerfile below.
 
 ```dockerfile
+# syntax=docker/dockerfile:1
```

Comment:
I removed the parameter from the `_config.yml`, and now have `dockerfile:1` hard-coded; I don't expect we'll reach v2 anytime soon, and if we do, a `find & replace` should be easy, so no need to use a parameter for it